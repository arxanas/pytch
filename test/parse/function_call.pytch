let foo =
  (bar baz) (qux grault)

let foo =
  bar baz qux grault
