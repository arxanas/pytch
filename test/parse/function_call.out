(Program
  (LetPlainStmt
    (Ident 'foo')
    (FunctionCallExpr
      (FunctionCallExpr
        (Ident 'bar')
        (Ident 'baz'))
      (FunctionCallExpr
        (Ident 'qux')
        (Ident 'grault'))))
  (LetPlainStmt
    (Ident 'foo')
    (FunctionCallExpr
      (Ident 'bar')
      (Ident 'baz')
      (Ident 'qux')
      (Ident 'grault'))))
